inc/Ui/Search.php

```diff
@@ -68,15 +68,15 @@ protected function getSearchFormHTML($query)
         $searchForm = (new Form())->attrs(['method' => 'get'])->addClass('search-results-form');
         $searchForm->setHiddenField('do', 'search');
         $searchForm->setHiddenField('id', $ID);
-        $searchForm->setHiddenField('searchPageForm', '1');
-        if ($INPUT->has('after')) {
-            $searchForm->setHiddenField('after', $INPUT->str('after'));
+        $searchForm->setHiddenField('sf', '1');
+        if ($INPUT->has('dta')) {
+            $searchForm->setHiddenField('dta', $INPUT->str('dta'));
```

Comment:
`dta` and `dtb`, this sound very cryptic... and b become chronologically before a, while alphabetically after.

So: for me these shortcuts are to cryptic... I would prefer the possibility to understand something from its name, this gives really no clue.

Reply:
We'll rename them to `min` and `max`